db/connection.ts

```diff
@@ -0,0 +1,32 @@
+import { drizzle } from "drizzle-orm/vercel-postgres";
+import { drizzle as drizzleNeon } from "drizzle-orm/neon-serverless";
+import { sql } from "@vercel/postgres";
+import { neon } from "@neondatabase/serverless";
+
+export function createDatabaseConnection() {
+  const neonUrl = process.env.DATABASE_URL || process.env.DATABASE_URL_UNPOOLED;
+  const vercelUrl = process.env.POSTGRES_URL;
```

Comment:
This is cool, but won't completely solve my problem. Let's simplify this all to just `const vercelUrl = process.env.POSTGRES_URL`, will need to do more research on how to generate and grab the preview DB url from vercel